SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -59,6 +59,10 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Replication", meta = (ConfigRestartRequired = false, DisplayName = "SpatialOS Network Update Rate"))
 	float OpsUpdateRate;
 
+	/** Replicate handover properties between servers, required for zoned worker deployments.*/
+	UPROPERTY(EditAnywhere, config, Category = "Replication", meta = (ConfigRestartRequired = false))
```

Comment:
Should we add DisplayName for this property?

Reply:
`bEnableHandover` becomes `Enable Handover` with the default conversion, which should be good enough?

Reply:
What does ConfigRestartRequired mean? Is it true that it's not required for any of the options?